### PR TITLE
Remove EI analytics docs

### PR DESCRIPTION
**Pull request by SanojPunchihewa**
## Purpose
Fixes https://github.com/wso2/docs-mi/issues/357

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -550,9 +550,6 @@ nav:
           - ELK Based Analytics:
               - Setup: mi-analytics/mi-elk-installation-guide.md
               - Access the Dashboard: mi-analytics/mi-elk-dashboards.md
-          - MI Analytics:
-              # - Setup: mi-analytics/setting-up-mi-analytics.md
-              - Access the Dashboard: mi-analytics/using-the-analytics-dashboard.md
```
Comment (chanikag) [inline en/mkdocs.yml:555]: Let's add redirects to Analytics overview page

File: en/mkdocs.yml [hunk 2]
```diff
@@ -1004,7 +1001,6 @@ plugins:
               'develop/hot-deployment.md': 'https://mi.docs.wso2.com/en/4.3.0/develop/using-remote-micro-integrator'
               'develop/using-wire-logs.md': 'https://mi.docs.wso2.com/en/4.3.0/develop/debugging-mediation'
               'install-and-setup/setup/deployment/mi-cicd-k8s.md': 'https://mi.docs.wso2.com/en/4.3.0/install-and-setup/setup/deployment/integration-cicd-overview'
-              'mi-analytics/setting-up-mi-analytics.md': 'https://mi.docs.wso2.com/en/4.3.0/observe-and-manage/observe-overview'
```
Comment (chanikag) [inline en/mkdocs.yml:1007]: Let's add redirects to Analytics overview page

File: en/mkdocs.yml (hunk 1 above)
Comment (SanojPunchihewa) [reply en/mkdocs.yml:555]: Fixed with [2133075](https://github.com/wso2/docs-mi/pull/948/commits/21330756e327fd8b4780e1b520d61a6edaf4b221)

File: en/mkdocs.yml (hunk 2 above)
Comment (SanojPunchihewa) [reply en/mkdocs.yml:1007]: Fixed with [2133075](https://github.com/wso2/docs-mi/pull/948/commits/21330756e327fd8b4780e1b520d61a6edaf4b221)

**Comment by chanikag** [comment]
Let's reword 
- https://mi.docs.wso2.com/en/latest/mi-analytics/mi-elk-installation-guide/

**Comment by SanojPunchihewa** [comment]
> Let's reword
> 
> * https://mi.docs.wso2.com/en/latest/mi-analytics/mi-elk-installation-guide/

Done with [345f4b5](https://github.com/wso2/docs-mi/pull/948/commits/345f4b5f048525c2314eee864cdcca3003def3a6)

**Review by chanikag — APPROVED** [review]

**Pull request merged**